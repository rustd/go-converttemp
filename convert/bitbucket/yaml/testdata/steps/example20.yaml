# Example using the caches option to cache dependencies for a Ruby project

definitions:
  caches:
    my-bundler-cache: vendor/bundle

pipelines:
  default:
    - step:
        caches:
          - my-bundler-cache # Cache is defined above in the definitions section
          - node # Pre-defined Pipelines cache
        script:
          - bundle install --path vendor/bundle
          - ruby -e 'print "Hello, World\n"'
