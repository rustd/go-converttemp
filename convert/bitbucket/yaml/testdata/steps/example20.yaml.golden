{
    "definitions": {
        "caches": {
            "my-bundler-cache": "vendor/bundle"
        }
    },
    "pipelines": {
        "default": [
            {
                "step": {
                    "caches": [
                        "my-bundler-cache",
                        "node"
                    ],
                    "script": [
                        "bundle install --path vendor/bundle",
                        "ruby -e 'print \"Hello, World\\n\"'"
                    ]
                }
            }
        ]
    }
}